src/connection.rs

```diff
@@ -180,7 +180,6 @@ pub enum ConnectionState {
 impl From<i64> for ConnectionState {
     fn from(state: i64) -> Self {
         match state {
-            0 => ConnectionState::Unknown,
```

Comment:
Clippy complained about the duplication with `_ => ConnectionState::Unknown,`, which is situated at the end of the match statement.